File: ec1/ec1.java
package ec1;

public class ec1 {

    private static int fibonacci(int n){
        if (n <= 1) {
            return n;
        }
        int output = 0;
        int x = 1;
        for (int i = 0; i < n; i++) {
            int temp = output;
            output = output + x;
            x = temp;
        }

        return output;
    }

    public static void main(String[] args) {
        String s = "";
        for (int i = 0; i < 10; i++) {
            s += fibonacci(i) + " ";
        }

        System.out.println(s);
    }
}

File: ec1/ec1.py
def fibonacci(n):
    if n <= 1:
        return n

    output = 0
    x = 1
    for i in range(0, n):
        temp = output
        output = output + x
        x = temp
    return output

string = ""
for i in range(0, 10):
    string = string + str(fibonacci(i)) + " "

print(string)